CuteLogger/include/AbstractAppender.h

```diff
@@ -24,22 +24,24 @@
 
 class CUTELOGGERSHARED_EXPORT AbstractAppender
 {
-  public:
```

Comment:
I can add an exception for the CuteLogger directory if we want.

Reply:
Yes, please exclude CuteLogger.

Reply:
Done